browser/brave_app_controller_mac.mm

```diff
@@ -15,21 +15,24 @@
 
 @implementation BraveAppController
```

Comment:
Now that this is compiled with ARC we should include the check at the top of the file: 

```objc
#if !defined(__has_feature) || !__has_feature(objc_arc)
#error "This file requires ARC support."
#endif
```

Reply:
Fixed in eccd437bb23f59d16d34a9cff773d542fdced514.

Reply:
Do we need a follow up issue to add this to all *_mac.mm files?

Reply:
@mkarolin We have an issue for it. - https://github.com/brave/brave-browser/issues/31372